### PR TITLE
[BEAM-5506] update create table to create external table

### PR DESCRIPTION
**Pull request by amaliujia**
Update `CREATE TABLE` to `CREATE EXTERNAL TABLE`.

Once your pull request has been opened and assigned a number, please edit the
URL below, replacing `PULL_REQUEST_NUMBER` with the number of your pull request.

http://apache-beam-website-pull-requests.storage.googleapis.com/562/index.html

Finally, it will help us expedite review of your Pull Request if you tag
someone (e.g. @username) to look at it.

### CODE REVIEW

**Comment by amaliujia** [comment]
reset this please

**Comment by amaliujia** [comment]
```
14:26:53 - ./.testcontent/documentation/dsls/sql/shell/index.html
14:26:53   *  internally linking to /documentation/dsls/sql/create-table/, which does not exist (line 305)
14:26:53      <a href="/documentation/dsls/sql/create-table/">CREATE EXTERNAL TABLE reference page</a>
14:26:53 - ./.testcontent/documentation/dsls/sql/statements/create-table/index.html
14:26:53   *  External link https://beam.apache.org/documentation/dsls/sql/create-external-table/ failed: 404 No error
```

@melap 

Can I get some advice on fix this 404 error?

File: src/documentation/dsls/sql/shell.md [hunk 1]
```diff
@@ -59,17 +59,17 @@ The shell converts the queries into Beam pipelines, runs them using `DirectRunne
...
+0: BeamSQL> CREATE EXTERNAL TABLE csv_file (field1 VARCHAR, field2 INTEGER) TYPE text LOCATION 'test-file.csv';
 
 No rows affected (0.042 seconds)
 ```
 
-The `CREATE TABLE` statement registers the CSV file as a table in Beam SQL and specifies the table's schema. This statement does not directly create a persistent physical table; it only describes the source/sink to Beam SQL so that you can use the table in the queries that read data and write data.
+The `CREATE EXTERNAL TABLE` statement registers the CSV file as a table in Beam SQL and specifies the table's schema. This statement does not directly create a persistent physical table; it only describes the source/sink to Beam SQL so that you can use the table in the queries that read data and write data.
 
-_For more information about `CREATE TABLE` syntax and supported table types, see the [CREATE TABLE reference page]({{ site.baseurl }}/documentation/dsls/sql/create-table/)._
+_For more information about `CREATE EXTERNAL TABLE` syntax and supported table types, see the [CREATE EXTERNAL TABLE reference page]({{ site.baseurl }}/documentation/dsls/sql/create-table/)._
```
Comment (melap) [inline src/documentation/dsls/sql/shell.md]: I think if you update this link to use create-external-table

File: src/documentation/dsls/sql/create-external-table.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: section
-title: "Beam SQL: CREATE TABLE Statement"
+title: "Beam SQL: CREATE EXTERNAL TABLE Statement"
 section_menu: section-menu/sdks.html
-permalink: /documentation/dsls/sql/create-table/
+permalink: /documentation/dsls/sql/create-external-table/
 redirect_from: /documentation/dsls/sql/statements/create-table/
```
Comment (melap) [inline src/documentation/dsls/sql/create-external-table.md]: hmm, try changing the redirects to also have the old create-table page. replace the line with this:

redirect_from:
  - /documentation/dsls/sql/statements/create-table/
  - /documentation/dsls/sql/create-table/

File: src/documentation/dsls/sql/shell.md (hunk 1 above)
Comment (melap) [inline src/documentation/dsls/sql/shell.md]: can update this link to use the -external also:  {{ site.baseurl }}/documentation/dsls/sql/create-external-table/

**Comment by melap** [comment]
Looks like that fixed the valid issues. this last one may be the mergebot bug (BEAM-3266) where it doesn't handle new files very well during testing. whoever approves may need to manually merge this PR.

**Comment by amaliujia** [comment]
R: @kennknowles @akedin @apilloud 
cc: @XuMingmin

**Comment by amaliujia** [comment]
Thanks for your help! @melap

**Comment by akedin** [comment]
LGTM

**Comment by amaliujia** [comment]
@melap

Could you help merge this PR?

**Comment by melap** [comment]
@amaliujia could you please squash your commits into one, and then I will merge. thanks!

**Comment by amaliujia** [comment]
@melap 

Done!

Thanks!

**Pull request merged**